client/src/Movies/MovieList.js

```diff
@@ -35,20 +37,24 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
```

Comment:
You can drop everything between movie-card, including the movie-card div itself, and replace all of it with <MovieCard /> with the necessary props and all
```js
function MovieDetails({ movie }) {
  const { title, director, metascore, stars } = movie;
  return (
        <MovieCard movie={movie}/>
  );
}
```